docs/source/quickstart.rst

```diff
@@ -275,3 +284,164 @@ At this point OHIF can be used to annotate the data in the DICOM server via the
 .. note::
 
   Here, user may also perform annotation using 3DSlicer by following the same instructions as in section :ref:`DeepEdit Annotation in 3DSlicer`.
+
+
+
+.. _Bundle:
+
+=============================================
+Tutorial 2 - Bundle App Tutorial and Use Cases
+=============================================
+
+Introduction:
+===============
+
+Customized Networks for MONAI Label
+------------------------------------
+
+This tutorial introduces the usage of the Bundle app in MONAILabel  - **monaibundle**.
+
+The Bundle App empowers MONAILabel with customized models, pre- and post-processing, and any anatomies for labeling tasks.
+The Bundle App supports various training/inference frameworks with `Model-Zoo <https://github.com/Project-MONAI/model-zoo>`_
```

Comment:
I don't quite understand this line, maybe better:
The Bundle App supports various [bundle](https://docs.monai.io/en/latest/mb_specification.html) based training / inference pipelines within the [Model-Zoo](https://github.com/Project-MONAI/model-zoo).

Thanks.